mailpoet/tests/acceptance/Segments/ManageSegmentsCest.php

```diff
@@ -172,7 +172,8 @@ public function deleteExistingSegment(\AcceptanceTester $i) {
     $i->wantTo('Empty trash from other segments');
     $i->waitForElementClickable('[data-automation-id="empty_trash"]');
     $i->click('[data-automation-id="empty_trash"]');
-    $i->waitForText('1 segment was permanently deleted.');
+    $i->waitForNoticeAndClose('1 segment was permanently deleted.');
+    $i->wait(10); // wait a moment for redirect loading
```

Comment:
This is the biggest wait in our acceptance tests. Do you think that ten seconds are necessary?
We could use a lower value and increase it when this test stays flaky. What do you think?

Reply:
I agree with you. Fixed [here](https://github.com/mailpoet/mailpoet/pull/4663/commits/58809db5534a2671d410db7a66e4cdc9929ad559)